packages/addons/service/snapclient/source/addon.py

```diff
@@ -9,9 +9,10 @@
 SNAPCLIENT = os.path.join(
     xbmcaddon.Addon().getAddonInfo('path'), 'bin', 'snapclient')
 
+line = ''
 card = ''
 cards = []
-lines = subprocess.check_output([SNAPCLIENT, '--list']).splitlines()
+lines = subprocess.getoutput(SNAPCLIENT + ' --list').splitlines()
```

Comment:
```suggestion
lines = subprocess.run([SNAPCLIENT, '--list'], capture_output=True, text=True).stdout.splitlines()
```
Python3 docu recommends using subprocess.run() when possible. Untested.